docroot/themes/humsci/humsci_basic/src/scss/elements/_tables.scss

```diff
@@ -0,0 +1,52 @@
+.hb-table-wrap {
+  margin: auto;
+  overflow-x: auto;
+}
+
+table {
+  $border-color: #d9d7d2;
+  // ⚠️ Do not set display: block; on a table ⚠️
+  // Tables need display: table; in order to semantically be a table
+  // display: block; will negatively affect screen reader a11y
+  width: 100%;
```

Comment:
make this a fixed min-width at smaller screen sizes

Reply:
actually, just need to add fixed `min-width: 150px` or similar to the column width

Reply:
Worked on this for a while. Adding a min-width on a column doesn't help. See general comments for more detail.

Reply:
ok, I'd be curious if @kaseybon could take a look tomorrow if we have a little time... but otherwise ok!